make-release.sh

```diff
@@ -57,6 +57,16 @@ bump_version () {
   git checkout "${CURRENT_BRANCH}"
 }
 
+function update_pkgs_versions() {
+  lerna version --no-git-tag-version -y "${VERSION}"
```

Comment:
TODO version  should be pass as argument since one place rely on VERSION, one on NEXT_VERSION